frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -71,10 +103,13 @@ const ProjectDetails: React.FC = () => {
           maxWidth={175}
         >
           <Stack hasGutter>
-            {sections.map(({ id, component }) => (
-              <StackItem key={id} id={id} aria-label={ProjectSectionTitles[id]}>
-                {component}
-              </StackItem>
+            {sections.map(({ id, component, isEmpty }, index) => (
+              <>
+                <StackItem key={id} id={id} aria-label={ProjectSectionTitles[id]}>
+                  {component}
+                </StackItem>
+                {index !== sections.length - 1 && isEmpty && <Divider />}
+              </>
```

Comment:
```suggestion
              <React.Fragment key={id}>
                <StackItem id={id} aria-label={ProjectSectionTitles[id]}>
                  {component}
                </StackItem>
                {index !== sections.length - 1 && isEmpty && <Divider />}
              </React.Fragment>
```

You should have the key at top of the fragment, if not react will not handle the rendering element key.